src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx

```diff
@@ -127,19 +126,25 @@ export const BuildBaronContent: React.VFC<BuildBaronCoreProps> = ({
       />
       {bbData?.searchReturnInfo?.issues.length > 0 && (
         <>
-          <TitleAndButtons>
+          <NonTableWrapper>
             {/* @ts-expect-error */}
             <TicketsTitle>
               Related tickets from Jira
               <StyledLink data-cy="jira-search-link" href={jiraSearchLink}>
                 {"  "}(Jira Search)
               </StyledLink>
             </TicketsTitle>
-          </TitleAndButtons>
+          </NonTableWrapper>
           {/* build baron related jira tickets */}
           <BuildBaronTable jiraIssues={bbData?.searchReturnInfo?.issues} />
         </>
```

Comment:
nit. Since everything else in this file has its own component, I kind of feel that this might also look nicer as a component

That way you wouldn't have to import stuff like `TicketsTitle`, `NonTableWrapper` into this file